src/drivers/intel/ace/ipc.c

```diff
@@ -169,5 +170,7 @@ int platform_ipc_init(struct ipc *ipc)
 	/* attach handlers */
 	intel_adsp_ipc_set_message_handler(INTEL_ADSP_IPC_HOST_DEV, message_handler, ipc);
 
+	irq_enable(PLATFORM_IPC_INTERRUPT);
```

Comment:
It will fail during compilation. PLATFORM_IPC_INTERRUPT isn't defined for mtl platform. Get interrupt number from device tree.

Reply:
> It will fail during compilation. PLATFORM_IPC_INTERRUPT isn't defined for mtl platform. Get interrupt number from device tree.

They are all in the soc\xtensa\intel_adsp\ace_v1x\include\ace_v1x-regs.h header (not yet in the devicetree) but retrieved them using proper macros 👍

Reply:
Have you ever tried compiling this code? :joy: